train.py

```diff
@@ -114,9 +114,14 @@
     # Checkpoint directory. Tensorflow assumes this directory already exists so we need to create it
     checkpoint_dir = os.path.abspath(os.path.join(out_dir, "checkpoints"))
     checkpoint_prefix = os.path.join(checkpoint_dir, "model")
+    lstm_checkpoint_prefix = os.path.join(checkpoint_dir, "lstm_model")
     if not os.path.exists(checkpoint_dir):
         os.makedirs(checkpoint_dir)
+
+    out1 = tf.placeholder(tf.float32, name="out1")
+    out2 = tf.placeholder(tf.float32, name="out2")
     saver = tf.train.Saver(tf.global_variables(), max_to_keep=2)
+    lstm_saver = tf.train.Saver([out1,out2], max_to_keep=100)
```

Comment:
lstm_saver = tf.train.Saver([siameseModel.out1,siameseModel.out2], max_to_keep=100)  ,
I guess this should work  instead of defining the placeholders and getting the values in the placeholders from sess.run.